src/data/pages/testing-multi-component.json

```diff
@@ -0,0 +1,121 @@
+{
+  "id": "expand4",
+  "investigation": "expanding-universe",
+  "order": "03",
+  "title": "Interpreting a Hubble Plot",
+  "slug": "testing-multi-component/",
+  "previous": {
+    "title": "Using Photometric Redshift to Measure Velocity",
+    "link": "/acquiring-the-data-2/"
+  },
+  "next": {
+    "title": "Interpreting a Hubble Plot",
+    "link": "/interpreting-a-hubble-plot-3/"
```

Comment:
So I think using these kinds of test pages is a totally helpful way to go about exploring these new features and getting stuff working in terms of the relationship between pageData and new components, but we don't have any decent way to not include a page like this in the whole investigation, but we don't really want it to be included in an investigation.  In addition, unless it has a unique ID it will potentially break another page that has the same ID (or, that is, I think one of them will not work).

Reply:
I understand and agree. I've removed the test page.